tag-historian/nitaghistorian.yml

```diff
@@ -228,26 +173,40 @@ definitions:
         format: iso-date-time
         example: '2018-09-04T18:45:08Z'
         default: '0001-01-01T00:00:00Z'
-      skip:
-        description: Limits the returned historical values to those after the given number of values.
-          Use with the take property to implement pagination. Only applies when not specifying a
-          decimation.
-        type: integer
-        minimum: 0
-        default: 0
       take:
         description: Limits the returned historical values to the specified number of values. Use
-          with the skip property to implement pagination. Only applies when not specifying a decimation.
+          with the continuationToken property to implement pagination. Only applies when not specifying a decimation.
         type: integer
         minimum: 0
         default: 0
         example: 1
+      continuationToken:
+        type: string
+        description: For non-decimated queries, a value generated by the server which should be sent
+          in the next request to retrieve the next page of values.
       decimation:
         description: When non-zero, the returned historical values are decimated across requested time
           span divided into the specified number of equal-sized intervals.
         type: integer
         minimum: 0
         default: 0
+      decimationStrategy:
```

Comment:
Let's remove this for now and keep it simple. We try to use the min/max strategy and don't support random sampling for now